roles/wildfly_validation/tasks/pidfile_validate.yml

```diff
@@ -0,0 +1,21 @@
+---
+- name: "PID file stat"
+  ansible.builtin.stat:
+    path: "{{ wildfly_validation_systemd_pidfile }}"
+  register: pid_file_stat
+
+- name: "Verify PID file existence"
+  ansible.builtin.fail:
+    msg: "PID file {{ wildfly_systemd.selinux.pid_path }}/{{ wildfly_instance_name }}.pid does not exist."
+  ignore_errors: true
+  when: not pid_file_stat.stat.exists
+  tags:
```

Comment:
why is this not idempotent? the pidfile should exist both in converge and idempotency stages